js/ui/scroll_view/ui.scrollbar.js

```diff
@@ -35,6 +35,7 @@ var Scrollbar = Widget.inherit({
             visibilityMode: SCROLLBAR_VISIBLE.onScroll,
             containerSize: 0,
             contentSize: 0,
+            isScrollUsedInsideOfScrollable: false,
```

Comment:
Is it possible to detect if scroll is used inside of scrollable without new option?

Reply:
It's possible to set default baseContainerSize and baseContentSize to impossible values e.g. -1, and detect if scroll is used inside of scrollable with these values.

Reply:
> Is it possible to detect if scroll is used inside of scrollable without new option?

It's possible to set default baseContainerSize and baseContentSize to impossible values e.g. -1, and detect if scroll is used inside of scrollable with these values.

Reply:
isScrollUsedInsideOfScrollable option is removed now. Is the current version ok?

Reply:
> Is it possible to detect if scroll is used inside of scrollable without new option?

isScrollUsedInsideOfScrollable option is removed now. Is the current version ok?